ms_game_chat/backend/backend/settings.py

```diff
@@ -40,7 +40,7 @@
 
 APPEND_SLASH = False
 
-ALLOWED_HOSTS = [os.environ.get('CURRENT_HOST'), 'backend', 'localhost']
+ALLOWED_HOSTS = [os.environ.get('CURRENT_HOST'), 'backend', 'localhost', '172.16.10.7']
```

Comment:
@mariekart42 i gave all container a fixed ip, this one is for the ms_auth service